src/php/Lookup/Response/Company.php

```diff
@@ -22,7 +22,8 @@ class Company
     const KEY_HOSTNAME  = 'Hostname';
     const KEY_DEBUG     = 'Debug';
     const KEY_TYPE      = 'Type';
-    const KEY_ORIGIP = "OrigIP";
+    const KEY_ORIGIP    = "OrigIP";
```

Comment:
Is 

const KEY_ORIGIP = "OrigIP";

Still being used - I think George removed this